server/public/model/post_test.go

```diff
@@ -995,3 +995,190 @@ func TestPostPriority(t *testing.T) {
 	p.Metadata.Priority.Priority = NewPointer(PostPriorityUrgent)
 	require.True(t, p.IsUrgent())
 }
+
+func TestPost_PropsIsValid(t *testing.T) {
```

Comment:
We are not adding tests to `post.IsValid`. I understand we are relying on the tests we already have for propsIsValid and attachment.IsValid.

0/5 if we should add some tests there for coverage / particular behavior of the `IsValid` function (like logging).

Reply:
For `post.IsValid`, nothing changes in this PR, right? 

I'm hesitant to add a test to `post.ValidateProps` as asserting the log output is a bit painful. Does such a test add much in terms of value?

Reply:
Short answer? No.

Long answer: They may add coverage (which is not a great metric, but a metric) and for completion, it verifies the behavior of "if something wrong happens, we indeed get a log so we can figure out what went wrong". That way, if someone mess with the log, the test will fail. But... the chance of something messing that is low.